<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Import</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>241</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_10">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Top</string>
           </property>
           <property name="margin">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboTop">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="editable">
            <bool>false</bool>
           </property>
           <property name="duplicatesEnabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnViewTop">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../../Desktop/images/icons/eye_of_horus.png</normaloff>../../Desktop/images/icons/eye_of_horus.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Base</string>
         </property>
         <property name="margin">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBase">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnViewBase">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../Desktop/images/icons/eye_of_horus.png</normaloff>../../Desktop/images/icons/eye_of_horus.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btnViewFilteredPointCloud">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>300</y>
      <width>181</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>View Filtered PointCloud</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../Desktop/images/icons/eye_of_horus.png</normaloff>../../Desktop/images/icons/eye_of_horus.png</iconset>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>181</width>
      <height>221</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Top</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>154</width>
        <height>151</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_15">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_15">
         <item>
          <widget class="QLabel" name="label_22">
           <property name="text">
            <string>Neighbours</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_13">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>5</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QSpinBox" name="spNbTop">
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="value">
            <number>6</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_16">
         <item>
          <widget class="QLabel" name="label_23">
           <property name="text">
            <string>Ratio</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_14">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QSpinBox" name="spRatioTop">
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="value">
            <number>10</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="btnStatFilterTop">
         <property name="text">
          <string>Run Statistical Filter</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnResetFilterTop">
         <property name="text">
          <string>Reset Filter</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QCheckBox" name="pcFilteredTop">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>170</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Show result</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Base</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>154</width>
        <height>151</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Neighbours</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>5</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QSpinBox" name="spNbBase">
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="value">
            <number>6</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Ratio</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QSpinBox" name="spRatioBase">
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="value">
            <number>10</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="btnStatFilterBase">
         <property name="text">
          <string>Run Statistical Filter</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnResetFilterBase">
         <property name="text">
          <string>Reset Filter</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QCheckBox" name="pcFilteredBase">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>170</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Show result</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Statistical Filter</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>20</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Radial Filter</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>40</y>
      <width>181</width>
      <height>221</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Top</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>154</width>
        <height>151</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_16">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_17">
         <item>
          <widget class="QLabel" name="label_24">
           <property name="text">
            <string>Neighbours</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_15">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>5</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QSpinBox" name="spNbTop_5">
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="value">
            <number>6</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_18">
         <item>
          <widget class="QLabel" name="label_25">
           <property name="text">
            <string>Ratio</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_16">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QSpinBox" name="spRatioTop_5">
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="value">
            <number>10</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="btnStatFilterTop_5">
         <property name="text">
          <string>Run Radial Filter</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnResetFilterTop_5">
         <property name="text">
          <string>Reset Filter</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QCheckBox" name="pcFilteredTop_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>170</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Show result</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Base</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>154</width>
        <height>151</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_17">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_19">
         <item>
          <widget class="QLabel" name="label_26">
           <property name="text">
            <string>Neighbours</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_17">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>5</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QSpinBox" name="spNbBase_3">
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="value">
            <number>6</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_20">
         <item>
          <widget class="QLabel" name="label_27">
           <property name="text">
            <string>Ratio</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_18">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QSpinBox" name="spRatioBase_3">
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="value">
            <number>10</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="btnStatFilterBase_3">
         <property name="text">
          <string>Run Radial Filter</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnResetFilterBase_3">
         <property name="text">
          <string>Reset Filter</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QCheckBox" name="pcFilteredBase_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>170</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Show result</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>241</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QPushButton" name="btnNearDistTop">
         <property name="text">
          <string>Calculate Nearest (Top)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnNearDistBase">
         <property name="text">
          <string>Calculate Nearest (Base)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QDoubleSpinBox" name="spDistValueTop">
         <property name="maximum">
          <double>3.500000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>3.500000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="spDistValueBase">
         <property name="maximum">
          <double>3.500000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>3.500000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>661</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btnViewPointCloudsImported">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>241</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>View Pointclouds</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../Desktop/images/icons/eye_of_horus.png</normaloff>../../Desktop/images/icons/eye_of_horus.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnViewPointCloudsDist">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>241</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>View Result</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../Desktop/images/icons/eye_of_horus.png</normaloff>../../Desktop/images/icons/eye_of_horus.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSavePointCloud">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>260</y>
      <width>181</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Pointcloud</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btnSample">
       <property name="text">
        <string>Sample</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spSample">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="singleStep">
        <number>5</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>261</width>
      <height>421</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <widget class="QCheckBox" name="chkTop">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>110</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Top</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkBase">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>110</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Base</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnConvexHull">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>160</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Convex Hull</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>9</y>
      <width>401</width>
      <height>391</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <widget class="QPushButton" name="btnToGeom">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>250</y>
       <width>181</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Generate Geometry</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>449</y>
      <width>671</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
   </widget>
   <zorder>frame_2</zorder>
   <zorder>frame</zorder>
   <zorder>tabWidget</zorder>
   <zorder>frame_3</zorder>
   <zorder>label</zorder>
   <zorder>verticalLayoutWidget_8</zorder>
   <zorder>btnViewFilteredPointCloud</zorder>
   <zorder>label_3</zorder>
   <zorder>label_15</zorder>
   <zorder>tabWidget_2</zorder>
   <zorder>horizontalLayoutWidget</zorder>
   <zorder>progressBar</zorder>
   <zorder>btnViewPointCloudsImported</zorder>
   <zorder>btnViewPointCloudsDist</zorder>
   <zorder>btnSavePointCloud</zorder>
   <zorder>horizontalLayoutWidget_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>692</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>file</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuabout">
    <property name="title">
     <string>about</string>
    </property>
   </widget>
   <addaction name="menufile"/>
   <addaction name="menuabout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
